bin/GameIndex.dbf

```diff
@@ -12859,6 +12859,7 @@ Serial = SLES-52931
 Name   = Legend of Kay
 Region = PAL-M5
 Compat = 5
+EETimingHack = 1		//For freezes in some scene transitions.
```

Comment:
Yea I know about this one , I just copied from the other version. Thought about changing it first but then I thought eh I'd leave it as it is to match the others.